packages/components/MenuButton/src/MenuButton.tsx

```diff
@@ -72,6 +72,7 @@ export const MenuButton = compose<MenuButtonType>({
       startIcon,
       componentRef: stdBtnRef,
       onClick: toggleShowContextualMenu,
+      accessibilityLabel,
```

Comment:
I wonder if we should just do something like ...props to avoid this happening in the future.

Reply:
That's a good suggestion and I don't see any issues with it. I think I'd prefer that.

Reply:
I'd agree. This should probably be the norm for our components moving forward. I'll change it

Reply:
Changed in newest commit